extension/jaegerremotesampling/internal/grpc_test.go

```diff
@@ -63,29 +63,44 @@ func TestSamplingGRPCServer_Shutdown(t *testing.T) {
 			grpcServer: &grpcServerMock{
 				timeToGracefulStop: time.Minute,
 			},
-			timeout: 5 * time.Second,
-		},
-		{
-			name:    "grpc server not started",
-			timeout: time.Minute,
-			expect:  errGRPCServerNotRunning,
+			timeout: time.Millisecond,
 		},
 	}
 
 	for _, tc := range tt {
 		t.Run(tc.name, func(t *testing.T) {
 			srv := &SamplingGRPCServer{grpcServer: tc.grpcServer}
 			ctx, cancel := context.WithTimeout(context.Background(), tc.timeout)
+			assert.NoError(t, tc.grpcServer.Serve(nil))
 			defer cancel()
 			assert.Equal(t, tc.expect, srv.Shutdown(ctx))
 		})
 	}
+
+	// Test what happens when the server was not started
+	srv := &SamplingGRPCServer{}
+	assert.Equal(t, errGRPCServerNotRunning, srv.Shutdown(context.Background()))
```

Comment:
It might be better to have this in its own test, that way a potential failure in the future will be more clear about which test is failing.

Reply:
Sure, I'll do